README.md

```diff
@@ -85,8 +86,13 @@ So you can use normal container parameter syntax.
 - service\_container is tested via simpletest for integration with Drupal. (run ./tests/run-simpletests.sh)
 - service\_container is tested via PHPUnit for code coverage. (run ./tests/run-coverage.sh)
 
-### Status
+### List of Drupal 8 services that we've altered
 
-[![Build Status](https://travis-ci.org/LionsAd/service_container.svg?branch=7.x-1.x)](https://travis-ci.org/LionsAd/service_container)
-[![Coverage Status](https://coveralls.io/repos/LionsAd/service_container/badge.png?branch=7.x-1.x)](https://coveralls.io/r/LionsAd/service_container?branch=7.x-1.x)
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/LionsAd/service_container?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Here's the list of the service that we've copied from Drupal 8.
+For some reasons, we had to alter them a bit to make them compatible with Drupal 7.
```

Comment:
Lets use:

The following files have been altered compared to their Drupal 8 variants to make them compatible with Drupal 7.